bin/node/runtime/build.rs

```diff
@@ -23,5 +23,7 @@ fn main() {
 		.with_wasm_builder_from_crates_or_path("2.0.0", "../../../utils/wasm-builder")
 		.export_heap_base()
 		.import_memory()
+		// Uncomment the following to enable wasm-based tracing
```

Comment:
I fought quite a bit before I found the proper syntax (undocumented the `\\\"`), so I thought it would be useful to keep it here to make it easier to activate for people building on it, but I sure can remove it.